azure-app-service/massdriver.yaml

```diff
@@ -78,16 +78,40 @@ params:
               const: P3v3
         minimum_worker_count:
           title: Minimum instances
-          description: Minimum number of instances for autoscale.
+          description: Minimum number of instances for autoscale. (Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
         maximum_worker_count:
           title: Maximum instances
-          description: Maximum number of instances for autoscale.
+          description: Maximum number of instances for autoscale.(Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
+      dependencies:
+        location:
+          oneOf:
+            - properties:
+                location:
```

Comment:
In Azure I've seen this as `azure_region`. Any reason for the change? Also, can we use the type like [here](https://github.com/massdriver-cloud/azure-virtual-network/blob/main/massdriver.yaml#L18)?

Reply:
I'm guessing this is because Azure capabilities are so segmented by region.